.github/workflows/test-provider.yaml

```diff
@@ -1,6 +1,9 @@
+# This is only workflow that requires cloud credentials and therefore will not run on PRs coming from forks.
 name: "Test Nebari Provider"
 
 on:
+  schedule:
+    - cron: "0 3 * * *"
```

Comment:
We need a mechanism to alert us in case of failures here. I just let a bot open an issue in case of failure. For example

[workflow](https://github.com/pmeier/light-the-torch/blob/20a75752e156651bb3dea2f497e1cde03690b8f4/.github/workflows/check-pytorch-package-indices.yml#L39-L47)

[local action](https://github.com/pmeier/light-the-torch/blob/20a75752e156651bb3dea2f497e1cde03690b8f4/.github/actions/publish-from-template/action.yml#L12-L25) with [Python script](https://github.com/pmeier/light-the-torch/blob/20a75752e156651bb3dea2f497e1cde03690b8f4/.github/actions/publish-from-template/render_template.py#L8-L12) (this is technically not needed, but believe me when I tell you I got bitten so often because the template failed to render and thus no issue was opened although the workflow failed)

[issue template](https://github.com/pmeier/light-the-torch/blob/20a75752e156651bb3dea2f497e1cde03690b8f4/.github/issue-templates/packages-out-of-sync.md?plain=1#L1-L16)

If we don't have something like that, any failure will go undetected until we have a PR from the main repo.

Reply:
I have added the `Test Nebari Provider` badge to the repo README so we can quickly check if the tests are failing. My main concern is that we're spending so much time maintaining these additional workflows for a minor benefit. 

We have a triage meeting once a week and it might be worth adding a "Validate CI is passing" task to those meetings.

Reply:
> My main concern is that we're spending so much time maintaining these additional workflows for a minor benefit.

Agreed. But what I proposed above just enhances visibility after we have killed the workflows from running on PRs.

> I have added the `Test Nebari Provider` badge to the repo README so we can quickly check if the tests are failing.

A red badge on README is a lot less visible than an issue. Are you committing to check this batch on a daily basis?